backend/db/simple_login_detials.txt

```diff
@@ -0,0 +1,13 @@
+Sample login details:
+
+-PATIENTS:
+name, surname, email, password
+Logan, Jackson, jackson@gmail.com, Jackson2044#
+Isabella, White, isabella_white@white.com, 999WhiteLike#FFFFFF
+Sophia, Brown, sophia.brown@op.pl, SophiaBrown098765432%
+
+-DOCTORS:
+name, surname, email, password
+Liam, Grimes, liam_grimes@op.pl, DontStopGrimes1346$
+James, Robinson, j_robin@son.com, TooMuchSameSurname0987654@
+Sophia, Johnson, soph.john@med.com, MedHere09765@
```

Comment:
Just a small info: I've allowed myself to edit this TXT into a md file so it's more readable :)